### PR TITLE
Update readme tips

### PR DESCRIPTION
**Pull request by poemusica**
Our current text is a bit vague and confusing for candidates.
Proposing an update.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -79,9 +79,9 @@ interview if you aren't able to solve these questions 😁.
 
 ## 💡 Tips for a Successful Interview
 
-- We conduct our interviews on [coderpad.io](https://coderpad.io/). It's similar
```
Comment (poemusica) [inline readme.md:82]: We might be switching to another platform in the future (i.e. HackerRank), so the proposed new wording removes mention of the exact platform we use.

File: readme.md [hunk 2]
```diff
@@ -95,5 +95,8 @@ interview if you aren't able to solve these questions 😁.
   what you don't understand.
 - Your interviewer wants to help you pass the interview. It may be helpful to
   think of them as a collaborator rather than a test proctor.
-- Technical interviews aren't like tests where you either pass or fail, so don't
-  worry if you falter or you don't solve the challenge completely.
+- Technical interviews help you and us determine whether you are ready to take
+  on the challenges of our Software Engineering course. In order to get the most
+  out of our program, we need to make sure you are comfortable with certain
+  programming basics including: basic data types (strings, integers, booleans,
+  lists), conditional logic, looping, and functions.
```
Comment (poemusica) [inline readme.md]: Could also add indexes/indices here?

**Comment by poemusica** [comment]
I also think the way that this section is worded is confusing
⬇️ ⬇️ ⬇️ 

image
⬆️ ⬆️⬆️
We should revise this message to be a bit more straightforward.

File: readme.md (hunk 2 above)
Comment (mearajennifer) [reply readme.md]: I think that makes sense. It's a specific area of lists that we want them to be aware of.

**Review by mearajennifer — COMMENTED** [review]
I like these changes!

File: readme.md (hunk 2 above)
Comment (poemusica) [reply readme.md]: cool, added indexing

**Review by mearajennifer — APPROVED** [review]

**Pull request merged**